### DISCUSSION TITLE
Need help with Tab remapping

### DISCUSSION THREAD

**Discussion by kaizerx31** [NvChad/NvChad#614] [Q&A] [answered]
hi, i just installed Nvchad and was trying to remap tab and it doesnt need to work for reason(other remaps are working fine), can anyone help ?
how i am remapping it
`hooks.add("setup_mappings", function(map) 
 map("n","<Tab>","dd",opt)
end)`

**Comment by siduck** [comment, marked as answer]
I think its just that tab is already used

**Reply by kaizerx31** [reply]
I dont understand , did i miss something with the map i have written?

**Reply by siduck** [reply]
> I dont understand , did i miss something with the map i have written?

oh sorry! I missed writing <TAB> , try that . btw whats dd?

**Reply by kaizerx31** [reply]
i was just checking if "dd" to delete line  is working to test if that remap is working, i tried the remap and it doesnt work. space and return work though

**Reply by siduck** [reply]
> i was just checking if "dd" to delete line is working to test if that remap is working, i tried the remap and it doesnt work. space and return work though

probably cuz TAB is used somewhere else , also you dont need to add opt in the mapping function!

try 
```
local hooks = require "core.hooks"

hooks.add("setup_mappings", function(map)
   map("n", "<leader>dd", "dd")
end)
```

**Reply by kaizerx31** [reply]
Thanks for the help, i just copied that opt thing from the docs. It works even without it

**Reply by siduck** [reply]
> Thanks for the help, i just copied that opt thing from the docs. It works even without it

I'll update the docs soon, it'll get easier to understand than before. Currently the major changes are been done in the dev branch.
